arrows/ocv/resection_camera.cxx

```diff
@@ -0,0 +1,198 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief OCV resection_camera algorithm implementation
+ */
+
+#include "resection_camera.h"
+
+#include <opencv2/calib3d/calib3d.hpp>
+#include <opencv2/core/eigen.hpp>
+#include <cmath>
+
+#include "camera_intrinsics.h"
+
+using namespace std;
+using namespace cv;
+using namespace kwiver::vital;
+
+namespace kwiver {
+namespace arrows {
+namespace ocv
+{
+
+/// private implementation
+struct resection_camera::priv
+{
+  priv()
+    : confidence_threshold(0.99)
```

Comment:
It looks like these parameter are left over from PNP.  We can remove them.  Later you may want to add parameters relating to which camera parameters to estimate, but these are not needed in the first pass.

When we do add parameters for controlling which camera parameters to estimate, we might want to borrow some of that configuration logic from the Ceres solver arrow, which does the same thing.

Reply:
changed to
    : reproj_accuracy(DBL_EPSILON)
    , max_iterations(30)
as specified in cv::calibrateCamera()